### PR TITLE
doc: Fix some typos at Formula Cookbook

### PR DESCRIPTION
**Pull request by marceloboeira**

### CODE REVIEW

File: share/doc/homebrew/Formula-Cookbook.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ so you can override this with `brew create <url> --set-name <name>`.
 
 ## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
 
-**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
+**We don’t accept formulas without [`homepages`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)!**
```
Comment (DomT4) [inline share/doc/homebrew/Formula-Cookbook.md]: `formulae` is actually correct here. It's the debate that will never die, but "formulae" is a valid plural of formula.

Comment (marceloboeira) [inline share/doc/homebrew/Formula-Cookbook.md]: I can actually restore the formulae, but there was a missing s on the "home pages" link ...

Comment (marceloboeira) [inline share/doc/homebrew/Formula-Cookbook.md]: @DomT4 fixed! Sorry about that!

File: share/doc/homebrew/Formula-Cookbook.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ so you can override this with `brew create <url> --set-name <name>`.
 
 ## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
 
-**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
+**We don’t accept formulae without [`homepages`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)!**
```
Comment (rakkesh) [inline share/doc/homebrew/Formula-Cookbook.md]: A formula has just one `homepage`; a less confusing way to address this is to add an `a` article prior to `homepage` - "We don’t accept formulae without a [`homepage`]"

Comment (marceloboeira) [inline share/doc/homebrew/Formula-Cookbook.md]: @rakkesh done!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9395b646744553eb1c38e07e338b2f213cfa25bd. Thank you for your contribution to Homebrew @marceloboeira; we appreciate it! :smiley_cat:

**Comment by marceloboeira** [comment]
@DomT4 Thanks! 😁

**Pull request closed without merging**